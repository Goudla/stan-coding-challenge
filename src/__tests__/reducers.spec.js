import { createStore } from 'redux';
import { createBrowserHistory } from 'history';

// Reducers
import createRootReducer from '../reducers';

// History
const history = createBrowserHistory();

const store = createStore(createRootReducer(history));

describe('reducers', () => {
  it('should be able to be used to configure store', () => {
    expect(store.getState()).toMatchSnapshot();
  });
});
