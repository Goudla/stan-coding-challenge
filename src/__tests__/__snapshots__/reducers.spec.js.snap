// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reducers should be able to be used to configure store 1`] = `
Object {
  "entries": Object {},
  "error": Object {},
  "initialDataLoaded": Object {},
  "loading": Object {},
  "router": Object {
    "action": "POP",
    "location": Object {
      "hash": "",
      "pathname": "/",
      "search": "",
      "state": undefined,
    },
  },
}
`;
